feature-libs/cart/import-export/core/model/import-export.model.ts

```diff
@@ -3,3 +3,10 @@ export type InvalidFileInfo = {
   invalidExtension?: Boolean;
   fileEmpty?: Boolean;
 };
+
+export type FileValidity = {
+  // size unit is MB
+  maxSize?: Number;
+  allowedExtensions?: string[];
+  checkEmptyFile?: Boolean;
```

Comment:
Any benefit of using Boolean object type here? I believe we expect a primitive `boolean` (lowercase) here?